row_test.go

```diff
@@ -146,7 +146,6 @@ func TestRowLoad(t *testing.T) {
 		filename string
 	}{
 		{"empty-two-cols", "testdata/empty-two-cols.dump"},
-		{"example", "testdata/example.dump"},
```

Comment:
?

Reply:
I didn't mean to push this, but it should be noted that while it's not passing locally, it passed on git.

Reply:
I have it fixed now, I noticed a comment in row_test.go that specifed that the positions may not match exactly, thought it might be good to fix the tests in the future I think this means my code passes